runtime/basilisk/src/benchmarking/route_executor.rs

```diff
@@ -37,6 +37,7 @@ pub const UNITS: Balance = 100_000_000_000;
 const MAX_NUMBER_OF_TRADES: u32 = 5;
 
 pub fn register_asset_with_name(name_as_bye_string: &[u8]) -> Result<AssetId, BenchmarkError> {
+	#![allow(clippy::result_large_err)]
```

Comment:
One small thing. I got the error two times. From `register_asset_with_name` and also from `generate_trades`. But this line catches both of them because the attribute is used with `!` after `#`. Then it is applied to a crate. Can we place it at the beginning of the file? Thanks. Some [info](https://stackoverflow.com/questions/25877285/how-to-disable-unused-code-warnings-in-rust)